src/dependency.json

```diff
@@ -0,0 +1,40 @@
+[
+	{
+		"url": "https://code.jquery.com/jquery-2.2.4.js",
+		"checksum": "sha256-iT6Q9iMJYuQiMWNd9lDyBUStIq/8PuOW33aOqmvFpqI="
+	},
+	{
+		"url": "https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.6.5/angular.js",
+		"checksum": ""
+	},
+	{
+		"url": "https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.0.0-beta/js/bootstrap.js",
+		"checksum": ""
+	},
+	{
+		"url": "https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.19.1/moment.js",
+		"checksum": ""
+	},
+	{
+		"url": "https://cdnjs.cloudflare.com/ajax/libs/Chart.js/1.0.2/Chart.js",
+		"checksum": ""
+	},
+	{
+		"url": "https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.7.1/js/bootstrap-datepicker.js",
+		"checksum": ""
+	},
+	{
+		"url": "https://cdnjs.cloudflare.com/ajax/libs/jqueryui/1.12.1/jquery-ui.js",
+		"checksum": ""
+	},
+	{
+		"url": "https://cdnjs.cloudflare.com/ajax/libs/foundation/6.4.3/js/foundation.js",
+		"checksum": ""
+	},{
+		"url": "https://cdnjs.cloudflare.com/ajax/libs/angular-ui-bootstrap/2.5.0/ui-bootstrap.js",
+		"checksum": ""
+	},{
+		"url": "https://cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore.js",
+		"checksum": ""
+	}
+]
```

Comment:
Hi, check indentation for this file, please. It should be 4 spaces, i believe it's indented in tabs.